<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>65</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtIP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConectar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Conectar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPorta">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>65</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Porta</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>65</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Intervalo</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderFaixa">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Faixa de valores</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtInicio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>65</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>a</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtFim">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConectar_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtLog">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>341</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>508</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
